README.md

```diff
@@ -114,38 +114,38 @@ This project aims to compare available knowledge management tools with regards t
-| Groups                |                                   | ✅                             |                                        | ✅                                   | ?                                      |                               | 4 fixed roles                              |                                                         |                                  |                                                      |                                  |
-| Passwordless sharing  | ✅ (identification via email)      | ✅ (identification via email)  | ✅ (identification via email)           | ❌                                   |                                        |                               |                                            |                                                         |                                  |                                                      |                                  |
-| Mulit-user            | ✅                                 | ✅                             | ✅                                      | ✅                                   | 🚧(sharing?)                            | ❌                             | ✅                                          | [❌](https://github.com/TriliumNext/Trilium/issues/4956) |                                  |                                                      |                                  |
+| public access (read)  | 🚧 no hierarchy                    | 🚧 no hierarchy                | ✅                                      | ✅                                   |                                        |                               | ✅                                         |                                                         |                                  |                                                      |                                  |
+| Public access (write) | ❌                                 | ❌                             | ❌                                      | ❌                                   |                                        |                               | ✅                                         |                                                         |                                  |                                                      |                                  |
+| Groups                |                                   | ✅                             |                                        | ✅                                   | ?                                      |                               | ✅                             |                                                         |                                  |                                                      |                                  |
```

Comment:
indeed, "Create New Role". Also here I will add the term "Role" to the description of this feature